collect_app/src/main/res/values/strings.xml

```diff
@@ -338,11 +338,11 @@
     <string name="get_trace">Start GeoTrace</string>
 
     <string name="geoshape_title">GeoShape</string>
-    <string name="select_geotrace_mode">"Select GeoTrace Mode"</string>
+    <string name="data_entry_mode">Data entry mode</string>
     <string name="start">Start</string>
     <string name="enable_gps">Enable GPS</string>
     <string name="geotrace_title">GeoTrace</string>
-    <string name="polygon_save">Save as Polygon</string>
+    <string name="polygon_save">Save as Polygon (This option will soon be removed!  See http://bit.ly/odk-polygon)</string>
```

Comment:
I would split this out into two strings so we don't trigger a translation of polygon_save. Maybe we can add a `\n\n` to separate this out. And perhaps something that is less vague than soon? 

> This option will be removed in next update. See https://forum.opendatakit.org/t/18277 for alternatives.

That ID is valid, by the way.

Reply:
Oh, that's weird.  When I go to https://forum.opendatakit.org/t/topic/18277, I get "Sorry, you don't have access to that topic!"

I was going to suggest that we start a forum topic just specifically about the disappearance of this button.  Is that what https://forum.opendatakit.org/t/topic/18277 is?

Reply:
You don't have access yet cause it's a scheduled post and you aren't a forum mod. Once we are close to release, we'll write it.

Reply:
Got it.  I've updated the message.

This is on a button, so the URL cannot be made a clickable link; and moreover the target audience, the form designer, will most likely view the forum page on a desktop or laptop computer.  So the URL will need to be retyped into a browser.  To make the message fit better inside the button and make the URL easier to retype, I set up a permanent bit.ly link that redirects to the forum page.

Here's how it looks:
![device-2019-03-08-052203](https://user-images.githubusercontent.com/236086/54030977-30a17980-4162-11e9-9841-603fe171866b.png)

How's that?